src/Interpreters/ProcessList.h

```diff
@@ -400,6 +400,7 @@ class ProcessList : public ProcessListBase
     /// Call under lock. Finds process with specified current_user and current_query_id.
     QueryStatusPtr tryGetProcessListElement(const String & current_query_id, const String & current_user);
 
+    /// Call under lock. Finds process with specified query_id.
```

Comment:
Can we use TSA annotations instead? They are much better than just comments

Reply:
Done in [`e1e2d07` (#66532)](https://github.com/ClickHouse/ClickHouse/pull/66532/commits/e1e2d07341afd30bbefcd9babc74f2832a2dc049)

Took much longer than I anticipated :sweat: